common/changes/pcln-design-system/UXPT-4810_2023-07-13-20-28.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "pcln-design-system",
+      "comment": "Added Tab component",
+      "type": "major"
```

Comment:
Not sure if supposed to be major or minor 😅

Reply:
Minor, please